rotkehlchen/serialization/deserialize.py

```diff
@@ -685,7 +685,12 @@ def deserialize_ethereum_address(symbol: str) -> ChecksumEthAddress:
     an ethereum address and is always checksummed. So also external input sanitization.
     https://github.com/rotki/rotki/issues/2334
     """
-    return ChecksumEthAddress(HexAddress(HexStr(symbol)))
+    try:
```

Comment:
You also have to rewrite the docstring. It has a huge TODO which you are solving with this issue.

Just describe what this function does (checks the given string, verifies it's a valid ethereum address and returns it)